crates/tlsn/src/verifier.rs

```diff
@@ -152,59 +152,6 @@ impl Verifier<state::Initialized> {
         })
```

Comment:
what about the other `notarize_with_provider` and `notarize` methods in `Verifier<state::Committed>`?